.github/workflows/release.yml

```diff
@@ -21,99 +21,195 @@ permissions:
-        BRANCH_COMMIT: ${{ inputs.branchCommit }}
-    - name: Cherry pick commits
-      run: |
-        commits=$(echo $CHERRYPICK_COMMITS | tr "," "\n")
-        for commit in $commits
-        do
-          echo "Cherry picking $commit."
-          git cherry-pick $commit
-        done
-      env:
-        CHERRYPICK_COMMITS: ${{ inputs.cherrypickCommits }}
-    - name: Add tag to most recent commit
-      run: |
-        DATE=$(date -d"next-monday - 1week" +'%Y-%m-%d')
-        T_COMMIT=$(git log -n 1 $RELEASE_BRANCH_NAME --pretty=format:'%H')
-        git tag -a $TAG_NAME -m "Release week of $DATE" $T_COMMIT
-      env:
-        RELEASE_BRANCH_NAME: ${{ steps.variables.outputs.releaseBranchName }}
-        TAG_NAME: ${{ steps.variables.outputs.tagName }}
-    - name: Setup Environment
-      id: setup-env
-      uses: ./.github/actions/setup-env
-    - name: Run Build
-      run: ./cicd/run-build
-    - name: Run Unit Tests
-      run: ./cicd/run-unit-tests
-    - name: Run Integration Smoke Tests
-      run: |
-        ./cicd/run-it-smoke-tests \
-        --it-region="us-central1" \
-        --it-project="cloud-teleport-testing" \
-        --it-artifact-bucket="cloud-teleport-testing-it-gitactions" \
-        --it-private-connectivity="datastream-private-connect-us-central1" \
-        --it-spanner-host="https://staging-wrenchworks.sandbox.googleapis.com/" \
-        --it-release=true \
-        --it-retry-failures=2
-    - name: Run Integration Tests
-      run: |
-        ./cicd/run-it-tests \
-        --it-region="us-central1" \
-        --it-project="cloud-teleport-testing" \
-        --it-artifact-bucket="cloud-teleport-testing-it-gitactions" \
-        --it-private-connectivity="datastream-private-connect-us-central1" \
-        --it-spanner-host="https://staging-wrenchworks.sandbox.googleapis.com/" \
-        --it-release=true \
-        --it-retry-failures=2
-    - name: Upload Tests Report
-      uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
-      if: always() # always run even if previous step fails
-      with:
-        name: surefire-test-results
-        path: '**/surefire-reports/TEST-*.xml'
-        retention-days: 1
-    - name: Create artifacts and push
-      run: |
-        mvn verify -PtemplatesRelease \
-        -DprojectId="dataflow-templates" \
-        -DbucketName="dataflow-templates-staging" \
-        -DlibrariesBucketName="dataflow-templates-libraries" \
-        -DstagePrefix="${CANDIDATE_NAME}" \
-        -Dmaven.test.skip -T8 -e
-      env:
-        CANDIDATE_NAME: ${{ inputs.candidateName }}
-    - name: Push tags
-      run: |
-        git push -u origin --tags
-    - name: Release
-      run: |
-        gh release create $TAG_NAME --title "Dataflow Templates $TAG_NAME" --notes ""
-      env:
-        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        TAG_NAME: ${{ steps.variables.outputs.tagName }}
+      - name: Get releaser identity
```

Comment:
Looks like there was an indentation change, which I think may break this workflow - could we revert? It will also make this section much easier to review

Reply:
Indentation change fixes format to match standard YAML format and matches that of the other workflows

https://yaml.org/spec/history/2003-09-01.html#:~:text=between%20YAML%20systems.-,5.1%20Sequence,-This%20type%20family

I reverted for easier review, but will probably re-indent before submit unless you have concerns.